### PR TITLE
fix some typo

### PR DESCRIPTION
**Pull request by qingsenLi**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/upgrade/common.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func getK8sVersionFromUserInput(flags *applyPlanFlags, args []string, versionIsM
 
 	// If the version is specified in config file, pick up that value.
 	if flags.cfgPath != "" {
-		// Note that cfg isn't preserved here, it's just an one-off to populate userVersion based on --config
+		// Note that cfg isn't preserved here, it's just a one-off to populate userVersion based on --config
```
Comment (SataQiu) [inline cmd/kubeadm/app/cmd/upgrade/common.go]: Thanks @qingsenLi 
But is it right? 
My dictionary tells me that `an one-off` is right.

File: pkg/controller/garbagecollector/garbagecollector.go [hunk 2]
```diff
@@ -401,7 +401,7 @@ func ownerRefsToUIDs(refs []metav1.OwnerReference) []types.UID {
 
 func (gc *GarbageCollector) attemptToDeleteItem(item *node) error {
 	klog.V(2).Infof("processing item %s", item.identity)
-	// "being deleted" is an one-way trip to the final deletion. We'll just wait for the final deletion, and then process the object's dependents.
+	// "being deleted" is a one-way trip to the final deletion. We'll just wait for the final deletion, and then process the object's dependents.
```
Comment (SataQiu) [inline pkg/controller/garbagecollector/garbagecollector.go]: My dictionary tells me that `an one-way` is right.

File: cmd/kubeadm/app/cmd/upgrade/common.go (hunk 1 above)
Comment (qingsenLi) [reply cmd/kubeadm/app/cmd/upgrade/common.go]: @SataQiu 
a before the consonant, an before the vowel, the word one is consonant pronunciation
I think so

**Comment by timothysc** [comment]
Closing.  

1 - This is not isolated to a given SIG 
2 - It is not provide meaningful value 
3 - You appear to be doing very minor spelling PRs which artificially can inflate you and your companies standing in the community. 
4 - Please work with the SIGs, on issues that exist in the kubernetes ecosystem.

**Pull request closed without merging**